lib/cli/cli.js

```diff
@@ -203,13 +199,14 @@ class CLI {
           }
         });
       });
+
+      onCommandInterrupt = () => Promise.resolve(command.onInterrupt()).then(() => runPromise);
```

Comment:
I have a slight preference to add a newline or two here, to make it easier to read...

```js
onCommandInterrupt = () => 
  Promise.resolve(command.onInterrupt())
    .then(() => runPromise);
```